src/test/java/com/amazon/opendistroforelasticsearch/sql/unittest/parser/SqlParserTest.java

```diff
@@ -545,6 +545,14 @@ public void indexWithDotsAndHyphen() throws SqlParseException {
         Assert.assertEquals("data-2015.08.22", select.getFrom().get(0).getIndex());
     }
 
+    @Test
+    public void systemIndexNameTest() throws SqlParseException {
```

Comment:
I think we could use a more descriptive test name here- something to indicate that we're testing an index with a dot character at the front of its name

Reply:
@davidcui-amzn I've changed it into `indexNameWithDotAtTheStart()`